plugins/idea/pluginImpl/src/main/java/com/imaginea/kodebeagle/impl/action/RefreshAction.java

```diff
@@ -21,43 +21,57 @@
 import com.imaginea.kodebeagle.base.object.WindowObjects;
 import com.imaginea.kodebeagle.base.util.UIUtils;
 import com.imaginea.kodebeagle.impl.util.JavaImportsUtil;
-import com.imaginea.kodebeagle.impl.util.ScalaImportsUtil;
 import com.intellij.openapi.editor.Editor;
 
+// import com.imaginea.kodebeagle.impl.util.ScalaImportsUtil;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)